File: NguGame/NguGame/NguGame/Models/Question.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NguGame.Models
{

    public class Question
    {
        public string nameQuestion { get; set; }
        public string rightAnswer { get; set; }
        public string wrongAnswer1 { get; set; }
        public string wrongAnswer2 { get; set; }
        public string wrongAnswer3 { get; set; }
        public string author { get; set; }
        public string explain { get; set; }
    }
}

File: appApi/Models/question.js
  // module.exports = User;
  const mongoose = require('mongoose');
  const Schema = mongoose.Schema;
  
  //Create Shema and models 
  
  const question = new Schema({
    nameQuestion:String,
    rightAnswer: String,
    wrongAnswer1:String,
    wrongAnswer2:String,
    wrongAnswer3:String,
    author:String,
    explain:String,
  });
  
  const Question = mongoose.model('ngu_game', question);
  
  module.exports = Question;
  
  